src/server/rpc/storage.rs

```diff
@@ -916,16 +916,15 @@ where
         })?
         .build();
 
-    let db = db_store
-        .db(&db_name)
-        .await
-        .context(DatabaseNotFound { db_name: &db_name })?;
+    let db = db_store.db(&db_name).await.context(DatabaseNotFound {
+        db_name: db_name.to_string(),
```

Comment:
I hope all these comments come across as helpful, not pedantic ❤️ 

```suggestion
        db_name: &*db_name,
```

Reply:
They're awesome! thanks 🥕

Reply:
🥕  indeed -- @carols10cents  I think has taught me 95% of what I know about Rust. Getting her comments on a PR is like a personal Rust tutor. 💯  thanks again @carols10cents